docs/utilities/parser.md

```diff
@@ -156,25 +156,26 @@ def my_function():
-| **KafkaSelfManagedEventModel**          | Lambda Event Source payload for self managed Kafka payload                   |
-| **KafkaMskEventModel**                  | Lambda Event Source payload for AWS MSK payload                              |
+| Model name                              | Description                                                                           |
+| --------------------------------------- | ------------------------------------------------------------------------------------- |
+| **AlbModel**                            | Lambda Event Source payload for Amazon Application Load Balancer                      |
+| **APIGatewayProxyEventModel**           | Lambda Event Source payload for Amazon API Gateway                                    |
+| **APIGatewayProxyEventV2Model**         | Lambda Event Source payload for Amazon API Gateway v2 payload                         |
+| **CloudwatchLogsModel**                 | Lambda Event Source payload for Amazon CloudWatch Logs                                |
+| **DynamoDBStreamModel**                 | Lambda Event Source payload for Amazon DynamoDB Streams                               |
+| **EventBridgeModel**                    | Lambda Event Source payload for Amazon EventBridge                                    |
+| **KafkaMskEventModel**                  | Lambda Event Source payload for AWS MSK payload                                       |
+| **KafkaSelfManagedEventModel**          | Lambda Event Source payload for self managed Kafka payload                            |
+| **KinesisDataStreamModel**              | Lambda Event Source payload for Amazon Kinesis Data Streams                           |
+| **KinesisFirehoseModel**                | Lambda Event Source payload for Amazon Kinesis Firehose                               |
+| **LambdaFunctionUrlModel**              | Lambda Event Source payload for Lambda Function URL payload                           |
+| **S3EventNotificationEventBridgeModel** | Lambda Event Source payload for Amazon S3 Event Notification to EventBridge.          |
+| **S3Model**                             | Lambda Event Source payload for Amazon S3                                             |
+| **S3ObjectLambdaEvent**                 | Lambda Event Source payload for Amazon S3 Object Lambda                               |
+| **S3SqsEventNotificationModel**         | Lambda Event Source payload for S3 event notifications wrapped in SQS event (S3->SQS) |
+| **SesModel**                            | Lambda Event Source payload for Amazon Simple Email Service                           |
+| **SnsModel**                            | Lambda Event Source payload for Amazon Simple Notification Service                    |
+| **SqsModel**                            | Lambda Event Source payload for Amazon SQS                                            |
```

Comment:
I took the opportunity to organize the names of the models in alphabetical order.